organizations.py

```diff
@@ -0,0 +1,150 @@
+
+    database_engine = None
+    database_config = {
+        "echo": True,
+        "pool_size": 5,
+        "max_overflow": 2,
+        "pool_timeout": 30,
+        "pool_recycle": 1800
+    }
+    secrets = {"CLOUD_SQL_CONNECTION_NAME": "", "DB_USER": "", "DB_PASSWORD": "", "PASS_PHRASE": ""}
+
+    def __init__(self):
+        """Initialize OrganizationManager."""
+
+        OrganizationManager.secrets = load_secrets(OrganizationManager.secrets)
+        # print(OrganizationManager.secrets["PASS_PHRASE"])
+
+        if app.config["IS_LOCAL"]:
+            self.database_engine = init_tcp_connection_engine(
+                OrganizationManager.secrets, Organization.database_name, OrganizationManager.database_config)
+        else:
+            self.database_engine = init_unix_connection_engine(
+                OrganizationManager.secrets, Organization.database_name, OrganizationManager.database_config)
+
+        Base.metadata.create_all(bind=self.database_engine)
+
+    def authenticate_organizations(self, login_id: str, login_password: str) -> int:
+        """Verify login.
+
+        Args:
+            login_id (str): Uniquely identifiable user ID registered with organizations.
+            login_password (str): Passwords for registered users in organizations.
+
+        Returns:
+            Int<ID or 0>: The integer value of the ID in the organization.
+        """
+
+        pass_phrase = OrganizationManager.secrets["PASS_PHRASE"]
+        session = Session(bind=self.database_engine)
+        authentications_statement = text(
+            "select id from organizations "
+            "where login_id = '%s' and AES_DECRYPT(login_password, SHA2('%s', 512)) = '%s'"
+            % (login_id, pass_phrase, login_password)
+        )
+        result = session.execute(authentications_statement).fetchone()
+        if (result is None):
+            return 0
+        else:
+            return result.id
+
+    def get_amazon_key(self, id: int) -> dict:
+        """Get Amazon Key.
+
+        Args:
+            id (int): ID of a registered user in organizations.
+
+        Returns:
+            Dict<Default Positions>: Dictionary containing the authentication key of Amazon.
+        """
+
+        amazon_dictionary = {}
+        session = Session(bind=self.database_engine)
+        pass_phrase = OrganizationManager.secrets["PASS_PHRASE"]
+        amazon_statement = text("select "
+                                "AES_DECRYPT(amazon_partner_tag, SHA2('%s', 512)) as amazon_partner_tag,"
+                                "AES_DECRYPT(amazon_access_key, SHA2('%s', 512)) as amazon_access_key,"
+                                "AES_DECRYPT(amazon_secret_key, SHA2('%s', 512)) as amazon_secret_key "
+                                "from organizations where id = %s"
+                                % (pass_phrase, pass_phrase, pass_phrase, id)
+                                )
+        amazon_result = session.execute(amazon_statement).fetchall()
```

Comment:
```suggestion
        amazon_result = session.execute(amazon_statement).fetchone()
```
にしておけば、108行目でforを使う必要もなくなりそうに思いました

Reply:
そうですね。
そうします。